securedrop/journalist_app/utils.py

```diff
@@ -507,40 +508,17 @@ def serve_file_with_etag(db_obj: Union[Reply, Submission]) -> flask.Response:
     return response
 
 
-class JournalistInterfaceSessionInterface(
-        sessions.SecureCookieSessionInterface):
-    """A custom session interface that skips storing sessions for api requests but
-    otherwise just uses the default behaviour."""
-    def save_session(self, app: flask.Flask, session: Any, response: flask.Response) -> None:
-        # If this is an api request do not save the session
-        if request.path.split("/")[1] == "api":
-            return
-        else:
-            super(JournalistInterfaceSessionInterface, self).save_session(
-                app, session, response)
-
-
-def cleanup_expired_revoked_tokens() -> None:
-    """Remove tokens that have now expired from the revoked token table."""
-
-    revoked_tokens = db.session.query(RevokedToken).all()
-
-    for revoked_token in revoked_tokens:
-        if Journalist.validate_token_is_not_expired_or_invalid(revoked_token.token):
-            pass  # The token has not expired, we must keep in the revoked token table.
-        else:
-            # The token is no longer valid, remove from the revoked token table.
-            db.session.delete(revoked_token)
-
-    db.session.commit()
+def logout_user(uid: int) -> None:
```

Comment:
This is user for logging out journalists when the passphrase is changed instead o using the nonce.